Sources/JExtractSwift/ConversionStep.swift

```diff
@@ -103,38 +76,30 @@ enum ConversionStep: Equatable {
 
   /// Convert the conversion step into an expression with the given
   /// value as the placeholder value in the expression.
-  func asExprSyntax(isSelf: Bool, placeholder: String) -> ExprSyntax {
+  func asExprSyntax(placeholder: String, bodyItems: inout [CodeBlockItemSyntax]) -> ExprSyntax? {
```

Comment:
Using `bodyItems`, `ConversionStep` can now introduce side effects, for example it can store the inner step into a variable, then return the `DeclReferenceExprSyntax` to it.